### ISSUE TITLE
breaks with express 4

### ISSUE THREAD

**Issue by dpouliot** [lgrcyanny/node-express-mysql-scaffolding#1]
I'm trying to update for express 4 but I'm getting errors I don't understand. 
express 4 no longer bundles middleware like compass. so I npm install compress and var compress = require('compress');, then it complains it can't find compress. probably something simple, but eluding me.

**Comment by lgrcyanny** [comment]
if can't find compress, you can add compress dependency in package.json, or you can specify the express version 3.4.7

**Comment by dpouliot** [comment]
compress has been renamed compression. 
http://scotch.io/bar-talk/expressjs-4-0-new-features-and-upgrading-from-3-0
I fixed all the bundled dependencies, and got the project to run, but it gave 404 (generic) errors that I couldn't figure out. :(

**Comment by lgrcyanny** [comment]
I am busy with another project recently, I will take care of this later on. Thanks for you feedback. You can check if the mysql config well, the sql has imported. and then make sure the 3000 port is available,  'npm start', visit 'localhost:3000', it works on my machine.

**Comment by dpouliot** [comment]
yes, I get to the site, the shell loads correctly (nav, footer, etc), and displays your custom 404. I see your code says you display 404 on any error. to be clear, I used a separate (though similar) database, but the 404 error happens when getting /login (never get to post), which makes me think the database is not the cause of the 404.

**Comment by lgrcyanny** [comment]
For the scaffolding, I only implement the user signin, signup function. The 404 and 500 error handler is in file config/express.js [line 94 - line 116].

**Comment by dpouliot** [comment]
I understand. I cannot get to /signup or /signin, they both 404
![image](https://cloud.githubusercontent.com/assets/2699333/3139902/89ce6ec2-e8f8-11e3-8f1a-7b6c3d3d2f87.png)
